src/qt/locale/bitcoin_fr.ts

```diff
@@ -1467,43 +1467,43 @@
     </message>
     <message>
         <source>Never share your private key to an untrustworthy person.</source>
-        <translation type="vanished">Ne partagez jamais votre clé privée avec une personne indigne de confiance.</translation>
+        <translation>Ne partagez jamais votre clé privée avec une personne indigne de confiance.</translation>
     </message>
     <message>
-        <source>Who owns the private keys owns the coins.</source>
-        <translation type="vanished">Ceux qui possèdent les clés privées possèdent les pièces.</translation>
+        <source>Who own the private keys own the coins.</source>
```

Comment:
If we change the source text the translation will not show, so this needs to be reverted to the source from 1.14.7-dev